packages/edit-site/src/components/global-styles/screen-style-variations.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import { isEqual } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { useMemo, useContext } from '@wordpress/element';
+import { ENTER } from '@wordpress/keycodes';
+import {
+	__experimentalGrid as Grid,
+	Card,
+	CardBody,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { mergeBaseAndUserConfigs } from './global-styles-provider';
+import { GlobalStylesContext } from './context';
+import StylesPreview from './preview';
+import ScreenHeader from './header';
+
+function compareVariations( a, b ) {
+	return isEqual( a.styles, b.styles ) && isEqual( a.settings, b.settings );
+}
+
+function Variation( { variation } ) {
```

Comment:
I think variations should support a name that is translatable. We are not providing any name or label describing what the variation is which is a bigger issue on screen readers.

Reply:
This has been discussed above, and it was suggested that it's better handled as a follow-up and not requirement for v1.

Reply:
Hi, is there an issue opened for this follow up?

Reply:
No, not right now, please consider opening one for more discussion. Thank you.